.github/workflows/continous-integration.yml

```diff
@@ -0,0 +1,183 @@
+#        repo-token: "${{ secrets.GITHUB_TOKEN }}"
+#        build-command: "make SPHINXOPTS=\"-W --keep-going -A html_theme=rasabaster\" html"
+#        pre-build-command: "apt-get update -y && apt-get install -y git"
+
+    - name: Test Docs 📃
+      run: |
+        pip install -r requirements-docs.txt
+        pip install -e .[sql]
+        cd docs && make SPHINXOPTS="-W --keep-going -A html_theme=rasabaster" html
+
+  test:
+    name: Run Tests
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 2
+      matrix:
+        python-version: [3.6, 3.7]
+
+    steps:
+    - name: Checkout git repository 🕝
+      uses: actions/checkout@v2
+
+    - name: Set up Python ${{ matrix.python-version }} 🐍
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Load Pip Cached Artifacts ⬇
+      uses: actions/cache@v1
+      with:
+        path: ~/.cache/pip
+        key: ${{ runner.os }}-${{ matrix.python-version }}-pip-${{ hashFiles('**/requirements-dev.txt') }}
+        restore-keys: |
+          ${{ runner.os }}-${{ matrix.python-version }}-pip-
+
+    - name: Install Dependencies 📦
+      run: |
+        sudo apt-get update -qq
+        python -m pip install -U 'pip<20'
+        pip install -r requirements-dev.txt
+        pip install -e .
+        make prepare-tests-ubuntu
+        pip list
+
+    - name: Test Code 🔍
+      run: |
+        make test
+
+    - name: Send Coverage Report 📊
+      if: matrix.python-version == 3.6
+      env:
+        COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_REPO_TOKEN }}
+        COVERALLS_SERVICE_NAME: github-ci
+      run: |
+        coveralls
+
+  deploy:
+    name: Deploy to PyPI
+    runs-on: ubuntu-latest
+
+    # deploy will only be run when there is a tag available
+    if: github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags')
+    needs: [quality, test, api]     # only run after all other stages succeeded
+
+    steps:
+    - name: Checkout git repository 🕝
+      uses: actions/checkout@v2
+
+    - name: Set up Python 3.6 🐍
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.6
+
+    - name: Build ⚒️ Distributions
+      run: |
+        python3 -m pip install --user --upgrade setuptools wheel
+        python setup.py sdist bdist_wheel
+
+    - name: Publish to PyPI 📦
+      uses: pypa/gh-action-pypi-publish@37e305e7413032d8422456179fee28fac7d25187
+      with:
+        user: __token__
+        password: ${{ secrets.PYPI_TOKEN }}
+
+    - name: Notify Slack & Publish Release Notes 🗞
+      env:
+        GH_RELEASE_NOTES_TOKEN: ${{ secrets.GH_RELEASE_NOTES_TOKEN }}
+        SLACK_WEBHOOK_TOKEN: ${{ secrets.SLACK_WEBHOOK_TOKEN }}
+        GITHUB_TAG: ${{ github.ref }}
+        GITHUB_REPO_SLUG: ${{ github.repository }}
+      run: |
+        sudo apt-get -y install pandoc
+        pip install -U github3.py pypandoc
+        python3 scripts/publish_gh_release_notes.py
```

Comment:
I think this is becoming more a matter of taste than facts. In the past using `python3` for anything that relies on python 3 was standard (because if you are not in an env, python might call a python 2.x interpreter). 

it's been like that on travis & makefile, as long as there are no problems and there is no python4 I'd suggest to keep it.

reference: https://www.python.org/dev/peps/pep-0394/#for-python-script-publishers